rest-api/requirements.txt

```diff
@@ -1,27 +1,66 @@
-# cloudstorage_api docs are at
-# cloud.google.com/appengine/docs/standard/python/googlecloudstorageclient/read-write-to-cloud-storage
+alembic==0.9.6
+aniso8601==1.3.0
```

Comment:
Is this just a reordering? How did this change occur?

You may want to revert this.

Reply:
done